contrib/roles/linux/kubernetes/tasks/prepare_master.yml

```diff
@@ -201,6 +202,7 @@
 
 - name: Kubernetes Master | Enabling OVN to listen on 6641 and 6642
   shell: |
+    set -o errexit
```

Comment:
Would make sense to remove the ``&&`` from next line in this case.

Reply:
Oh, I didn't notice the `&&`. I'll remove the `set -o errexit` since it's useless here.